lib/rules/array-bracket-newline.js

```diff
@@ -15,8 +15,26 @@ const astUtils = require("./utils/ast-utils");
 /** @type {import('../shared/types').Rule} */
 module.exports = {
     meta: {
-        deprecated: true,
-        replacedBy: [],
+        deprecated: {
+            message: "Formatting rules are being moved out of ESLint core.",
+            url: "https://eslint.org/blog/2023/10/deprecating-formatting-rules/",
+            deprecatedSince: "8.53.0",
+            availableUntil: "10.0.0",
+            replacedBy: [
+                {
+                    message: "ESLint Stylistic now maintains deprecated stylistic core rules",
```

Comment:
There's an inconsistency between the top-level `message`, which ends with a period, and this one, which does not. I think we should be consistent and have messages end with periods throughout.